deployment.rst

```diff
@@ -101,10 +101,10 @@ specifically tailored to the requirements of Symfony.
     `Symfony plugin`_ is a plugin to ease Symfony related tasks, inspired by `Capifony`_
     (which works only with Capistrano 2).
 
-Common Post-Deployment Tasks
-----------------------------
+Common Pre-Post-Deployment Tasks
```

Comment:
Please add a manual anchor to keep it working like with the headline before.

I am currently on a phone 📱 

@javiereguiluz could you please add this anchor while merging? Thanks 😊